superset-frontend/packages/superset-ui-chart-controls/src/sections/advancedAnalytics.tsx

```diff
@@ -121,6 +121,10 @@ export const advancedAnalyticsControls: ControlPanelSectionConfig = {
             ['2 years ago', t('2 years ago')],
             ['156 weeks ago', t('156 weeks ago')],
             ['3 years ago', t('3 years ago')],
+            ['208 weeks ago', t('208 weeks ago')],
```

Comment:
It seems like the Y4Y comparison was removed in https://github.com/apache/superset/pull/22150, though it's not evident—per the PR description and/or comments—as to why, hence I re-added them.